build/Dockerfile

```diff
@@ -3,11 +3,7 @@ FROM registry.access.redhat.com/ubi8/go-toolset as builder
 
 WORKDIR /go/src/github.com/openshift/compliance-operator
 
-ARG ARG_GOPROXY="https://proxy.golang.org"
-ARG ARG_GOSUMDB="sum.golang.org"
-
-ENV GOPROXY=$ARG_GOPROXY
-ENV GOSUMDB=$ARG_GOSUMDB
+ENV GOFLAGS=-mod=vendor
```

Comment:
Don't we also need GO111MODULE=auto here?

Reply:
GO111MODULE=auto is kind of an overkill really. It already is the default.